docs/readme.html

```diff
@@ -243,8 +245,6 @@
   pc.setLocalDescription(myOffer);
   ```
 
-  _setLocalDescription_ indirectly controls the candidate gathering process.
```

Comment:
@ggarber, good news for M57 :)

https://www.chromestatus.com/feature/4973817285836800
https://bugs.chromium.org/p/chromium/issues/detail?id=673395
https://www.w3.org/TR/webrtc/#dom-rtcconfiguration-icecandidatepoolsize